docs/raw/serving-community-history.md

```diff
@@ -53,19 +53,24 @@ The following terminology is used throughout this specification:
 
 | Name                 | References |
 | -------------------- | --- |
-| Waku node            | An Ethereum node with Waku V1 enabled, or a [10/WAKU2](https://rfc.vac.dev/spec/10/) node that implements [11/WAKU2-RELAY](https://rfc.vac.dev/spec/11/)|
+| Waku node            | An Waku node that implements [6/WAKU1](https://rfc.vac.dev/spec/6/), or a [10/WAKU2](https://rfc.vac.dev/spec/10/) node that implements [11/WAKU2-RELAY](https://rfc.vac.dev/spec/11/)|
 | Store node           | A Waku node that implements [11/WAKU-MAILSERVER](/spec/11) or [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/) respectively |
-| Waku network         | A group of Waku nodes connected through the internet connection and forming a graph |
-| Community owner      | A Status user that owns a Status community |
-| Community member     | A Status user that is part of a Status community |
+| Waku network         | A group of Waku nodes forming a graph, connected via [RLPx transport protocol](https://rfc.vac.dev/spec/6/#wire-specification) or [11/WAKU2-RELAY](https://rfc.vac.dev/spec/11/) respectively |
+| Status user          | An Ethereum account that can be used in a Status consumer product, such as Status Mobile or Status Desktop |
+| Status node          | A Status client run by a Status application |
+| Community owner      | A Status user that owns the private key for a Status community |
+| Community member     | A Status user that is part of a Status community, not owning the private key of the community |
 | Community owner node | A Status node with message archive capabilities enabled, run by a community owner |
 | Community member node| A Status node with message archive capabilities enabled, run by a community member |
 | Live messages        | Waku messages received through the Waku network |
+| BitTorrent client    | A program implementing the [BitTorrent](https://bittorrent.org) protocol |
+| Torrent/Torrent file | A file containing metadata about data to be downloaded by BitTorrent clients |
+| Magnet link          | A link encoding the metadata provided by a torrent file ([Magnet URI scheme](https://en.wikipedia.org/wiki/Magnet_URI_scheme)) |
 
 
 ## Supported platforms
 
-Creating, distributing, downloading and unpacking community history archives SHOULD only be supported in desktop clients. Mobile clients SHOULD NOT implement this functionality due to bandwidth and storage limitations.
+Creating, distributing, downloading and unpacking community history archives SHOULD only be supported in Status desktop clients. Status mobile clients SHOULD NOT implement this functionality due to bandwidth and storage limitations.
```

Comment:
I would change this to: Status mobile are out of scope for the MVP of this service.  However we may consider bringing this service to Status Mobile in the future.

Thinking behind this comment: in many countries mobile bandwidth is plentiful, very fast, and very cheep, for example in Ukraine it costs about $5 per month for unlimited mobile data, and the speed of this mobile data is normally between 50 and 100 megabits per second (faster than wired ADSL in London!).  In the US mobile data is both expensive and slow, but for folks who are in countries with fast a cheep mobile data, mobile data usage isn't a worry.

Re. storage, in 64gb seems to be becoming the storage baseline these days, with the latest top of the line phones now offering 1TB storage(!!!).  So storage will probably not be a limitation on mobile in the future.

Reply:
Happy to change that!